pixart.js

```diff
@@ -1 +1,30 @@
-
+var button = document.querySelector("#set-color");
+var color_field = document.querySelector("#color-field");
+var brush = document.querySelector(".brush");
+button.addEventListener("click", function()
+  {
+    event.preventDefault();
+    brush.style.background = color_field.value;
+  });
+color_field.addEventListener("keypress", function(e){
+  if (13 == e.keyCode) {
```

Comment:
Can't think of any other way to do it! But I like `e.keyCode == 13` instead of the other way around.